resdk/shortcuts/collection.py

```diff
@@ -65,7 +65,7 @@ def _create_relation(self, relation_type, samples, positions=[], label=None):
         if label:
             relation_data['label'] = label
 
-        self.resolwe.relation.create(**relation_data)
+        return self.resolwe.relation.create(**relation_data)
```

Comment:
Similar as above, would this change/fix be worth mentioning in the commit message?